src/ScreenManager.cpp

```diff
@@ -80,7 +80,7 @@ static Preference<bool> g_bDelayedScreenLoad( "DelayedScreenLoad", false );
 //static Preference<bool> g_bPruneFonts( "PruneFonts", true );
 
 // Screen registration
-static std::map<RString,CreateScreenFn>	*g_pmapRegistrees = nullptr;
+static std::unordered_map<std::string,CreateScreenFn>	*g_pmapRegistrees = nullptr;
```

Comment:
You're the one that changed them to use nullptr, so that's on you.  This PR is just about changing from map to unordered_map.